tokens/eth/0xba100000625a3754423978a60c9317c58a424e3D.json

```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "BAL",
+  "name": "Balancer",
+  "type": "ERC20",
+  "address": " 0xba100000625a3754423978a60c9317c58a424e3D",
```

Comment:
Please remove the space in the beginning

Reply:
whoops. Thanks!